src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;

```

Comment:
Consider changing the String 'fruit' to a Fruit object if there is a Fruit class. This would allow for better object-oriented design.

Reply:
ok, done